### PR TITLE
Cypress testing (& some minor privacy fixes)

### PR DESCRIPTION
**Pull request by eamyzzaj**
# Details #
Further description of things changed. Please feel free to test my changes, especially for entries, before approving/merging.

## MAIN:  ##
- Cypress Testing is now in effect!! It tests the main deployed app. If you want to test local changes, just need to change visit urls in .cy.js files to be the localhost address. Only currently has tests for login and some of diary entry. 

## MINOR:  ##
- Changed some minor logic for diary entries. Previously, private saved entries would not have a published date which would automate their date to December 1969. Now, private entries will save as published always and always have a publish date. 
- I changed the schema for the public_entries view in Supabase a little so now it only pulls from entries where is_public is TRUE. Public scribbles page now should not feature any private entries.

## Upcoming:  ##
I will implement more tests and also create one large one for a "happy workflow".

### CODE REVIEW

File: packages/express-backend/routes/entries.js
Comment (eamyzzaj) [inline packages/express-backend/routes/entries.js:1]: I just commented out this line cause it wasn't being used 
I can fully delete it later

File: packages/express-backend/routes/entries.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ router.get("/:entry_id", async (req, res) => {
 
 //used to look up all public entries
 router.get("/", async (req, res) => {
-  const is_public = req.query.is_public ? req.query.is_public : "False";
+  // const is_public = req.query.is_public ? req.query.is_public : "False";
```
Comment (eamyzzaj) [inline packages/express-backend/routes/entries.js:48]: I just commented out this line cause it wasn't being used
I can fully delete it later

We handle this logic through the VIEW table in supabase

File: packages/react-frontend/src/DiaryEntry.jsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ function DiaryEntry(props) {
       title,
       entry,
       is_public: isPrivate ? "false" : "true",
-      status: isPrivate ? "draft" : "published",
+      status: "published",
```
Comment (eamyzzaj) [inline packages/react-frontend/src/DiaryEntry.jsx:74]: Everything is always "published" now, no drafts

**Review by Mcray3000 — APPROVED** [review]
Testing works, and the changes look good. Do you know if this is already in the workflow file?

**Review by TLMatthies — APPROVED** [review]
LGTM!

**Review by Alaxxn — APPROVED** [review]

**Pull request closed without merging**